lib/Options.php

```diff
@@ -40,10 +43,16 @@ class Options {
 
     private $shutdownTimeout = 3000; // milliseconds
 
+    private $_dynamicCache;
+
     private $_initialized = false;
 
     public function __construct() {
         $this->deflateEnable = \extension_loaded("zlib");
+
+        $this->_dynamicCache = new class {
+            public $deflateContentTypes = [];
+        };
```

Comment:
Uhm, I mean that __set() is preventing direct writes after initialization.
And I didn't feel like adding an explicit exception for _dynamicCache right there.